src/omd.ml

```diff
@@ -14,6 +14,41 @@ let parse_inlines (md, defs) =
   in
   List.map (Mapper.map (parse_inline defs)) md
 
+let txt ?(attrs = []) s = Text (attrs, s)
+
+let em ?(attrs = []) il = Emph (attrs, il)
+
+let strong ?(attrs = []) il = Strong (attrs, il)
+
+let code ?(attrs = []) s = Code (attrs, s)
+
+let hard_break attrs = Hard_break attrs
+
+let soft_break attrs = Soft_break attrs
```

Comment:
So long as these constructors live in the `Omd` module (rather than the lower-level `Ast` module), I *think* we'd never have any need for attributes on the breaks? So perhaps these should just be:

```suggestion
let hard_break = Hard_break []

let soft_break = Soft_break []
```

What do you think?